system-parachains/collectives/collectives-polkadot/src/ambassador/mod.rs

```diff
@@ -0,0 +1,358 @@
+	pub const ProposalBond: Permill = Permill::from_percent(100);
+	pub const Burn: Permill = Permill::from_percent(0);
+	pub const MaxBalance: Balance = Balance::max_value();
+	// The asset's interior location for the paying account. This is the Ambassador Treasury
+	// pallet instance.
+	pub AmbassadorTreasuryInteriorLocation: InteriorLocation =
+		PalletInstance(<crate::AmbassadorTreasury as PalletInfoAccess>::index() as u8).into();
+}
+
+#[cfg(feature = "runtime-benchmarks")]
+parameter_types! {
+	pub const ProposalBondForBenchmark: Permill = Permill::from_percent(5);
+}
+
+/// [`PayOverXcm`] setup to pay the Ambasssador Treasury.
+pub type AmbassadorTreasuryPaymaster = PayOverXcm<
+	AmbassadorTreasuryInteriorLocation,
+	crate::xcm_config::XcmRouter,
+	crate::PolkadotXcm,
+	ConstU32<{ 6 * HOURS }>,
+	VersionedLocation,
+	VersionedLocatableAsset,
+	LocatableAssetConverter,
+	VersionedLocationConverter,
+>;
+
+pub type AmbassadorTreasuryInstance = pallet_treasury::Instance2;
+
+impl pallet_treasury::Config<AmbassadorTreasuryInstance> for Runtime {
+	// The creation of proposals via the treasury pallet is deprecated and should not be utilized.
+	// Instead, public or fellowship referenda should be used to propose and command the treasury
+	// spend or spend_local dispatchables. The parameters below have been configured accordingly to
+	// discourage its use.
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type ApproveOrigin = frame_support::traits::NeverEnsureOrigin<Balance>;
+	#[cfg(feature = "runtime-benchmarks")]
+	type ApproveOrigin = EnsureRoot<AccountId>;
+	type OnSlash = ();
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type ProposalBond = ProposalBond;
+	#[cfg(feature = "runtime-benchmarks")]
+	type ProposalBond = ProposalBondForBenchmark;
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type ProposalBondMinimum = MaxBalance;
+	#[cfg(feature = "runtime-benchmarks")]
+	type ProposalBondMinimum = ConstU128<{ ExistentialDeposit::get() * 100 }>;
+	#[cfg(not(feature = "runtime-benchmarks"))]
+	type ProposalBondMaximum = MaxBalance;
+	#[cfg(feature = "runtime-benchmarks")]
+	type ProposalBondMaximum = ConstU128<{ ExistentialDeposit::get() * 500 }>;
+	// end.
+
+	type WeightInfo = weights::pallet_treasury_ambassador_treasury::WeightInfo<Runtime>;
+	type RuntimeEvent = RuntimeEvent;
+	type PalletId = AmbassadorTreasuryPalletId;
+	type Currency = Balances;
+	type RejectOrigin = OpenGovOrHeadAmbassadors;
+	type SpendPeriod = ConstU32<{ 7 * DAYS }>;
+	type Burn = Burn;
+	type BurnDestination = ();
+	type SpendFunds = ();
+	type MaxApprovals = ConstU32<100>;
+	type SpendOrigin = EitherOf<
+		EitherOf<
+			EnsureRootWithSuccess<AccountId, MaxBalance>,
+			MapSuccess<
+				EnsureXcm<IsVoiceOfBody<GovernanceLocation, TreasurerBodyId>>,
+				Replace<ConstU128<{ 10_000 * GRAND }>>,
+			>,
+		>,
+		EitherOf<
+			MapSuccess<SeniorAmbassadors, Replace<ConstU128<{ 1 * GRAND }>>>,
```

Comment:
in manifesto the spend limit is mentioned in USD, but they should be in DOT, here I am setting an equivalent value in DOT.

Reply:
I would still try to keep the values roughly aligned.
```suggestion
			MapSuccess<SeniorAmbassadors, Replace<ConstU128<{ 100 * UNITS }>>>,
```